lib/active_merchant/billing/gateways/adyen.rb

```diff
@@ -622,9 +623,11 @@ def add_mpi_data_for_network_tokenization_card(post, payment, options)
 
         post[:mpiData] = {}
         post[:mpiData][:authenticationResponse] = 'Y'
-        post[:mpiData][:cavv] = payment.payment_cryptogram
         post[:mpiData][:directoryResponse] = 'Y'
         post[:mpiData][:eci] = payment.eci || '07'
```

Comment:
Yikes. This default of 07 is only applicable to Visa but we also support NT for Mastercard. We should change the default depending on the card scheme. 

I think we have (had?) a ticket for that somewhere

Reply:
Oh really, let me look it up on their docs and also look for that ticket

Reply:
It's not an Adyen thing/default it's an ECI value as a concept. IIRC all Mastercard NTs are failing right now on this gateway

Reply:
Do you know what the standard values are? I can't see anything in their docs for defaults and I still haven't heard back from Adyen yet

Reply:
Just emailed you the message from Mastercard